code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm

```diff
@@ -186,7 +186,7 @@
 /datum/crafting_recipe/food/royalcheese
 	name = "Royal Cheese"
 	reqs = list(
-		/obj/item/food/cheese/wheel = 1,
+		/obj/item/food/cheese = 1,
```

Comment:
Also, this includes subtypes, so this makes it so you can craft royal cheese with cheese slices. I don't know of an easy solution to this problem.